bitcoin/script.c

```diff
@@ -214,6 +215,14 @@ u8 *scriptpubkey_p2pkh(const tal_t *ctx, const struct bitcoin_address *addr)
 	return script;
 }
 
+u8 *scriptpubkey_opreturn(const tal_t *ctx)
+{
+	u8 *script = tal_arr(ctx, u8, 0);
+
+	add_op(&script, OP_RETURN);
```

Comment:
Does not standard `OP_RETURN` require an additional push after the `OP_RETURN`?  Otherwise the tx may not get broadcast across the Bitcoin network to reach a miner?

Reply:
bitcoind source doesn't seem to, but I added a test to be sure!

Reply:
Where is test? I think regtest and testnet skip over `IsStandardTx` check?

The bitcoind source doesn't seem to indeed; it allows a plain `OP_RETURN`: https://github.com/bitcoin/bitcoin/blob/d889c036cd6f683116e6a27e404be2809d1deb76/src/script/standard.cpp#L88-L96

All the guides I found on the Internet indicate a single data push after `OP_RETURN` but the actual code seems to allow 0 to any number of data pushes, as long as the total length is below the limit.